meta-resin-common/recipes-support/resin-device-register/resin-device-register/resin-device-register

```diff
@@ -62,7 +57,7 @@ while true; do
         if [ "$status_code" -eq 201 ]; then
             device_id=$(jq -r '.id' /var/log/register-device-curl.log)
             config_json=`cat $CONFIG_PATH`
-            echo $config_json | jq ".uuid=\"$uuid\"|.registered_at=\"`date +%s`\"|.deviceId=\"$device_id\"" > $CONFIG_PATH
+            echo $config_json | jq ".registered_at=\"`date +%s`\"|.deviceId=\"$device_id\"" > $CONFIG_PATH
```

Comment:
I know that, but if you decide to remove it from here, there is no point in reading it above either, is it?

`read api_key user_id application_id device_type uuid <<<$(jq -r '.apiKey,.userId,.applicationId,.deviceType // "raspberry-pi",.uuid' $CONFIG_PATH)`